genesis/process/sovereignGenesisBlockCreatorFactory_test.go

```diff
@@ -0,0 +1,26 @@
+//go:build !race
```

Comment:
Still need this line?

Reply:
Yes, since I use `createMockArgument` func from the same test package that is being built like this. Otherwise, it wouldn't build